### ISSUE TITLE
PixieEngine.com - online pixel editor

### ISSUE THREAD

**Issue by STRd6** [opensourcerails/opensourcerails#23]
PixieEngine is an online pixel editor and game creation site where people can create and share pixel art.

https://github.com/PixieEngine/pixieengine.com

**Comment by mbajur** [comment]
Is that project still alive?

**Comment by STRd6** [comment]
Yeah, it's moved here: https://github.com/STRd6/pixie2, live at http://pixieengine.com/

**Comment by railsjedi** [comment]
pixieengine project looks great!

**Comment by mbajur** [comment]
:+1:
